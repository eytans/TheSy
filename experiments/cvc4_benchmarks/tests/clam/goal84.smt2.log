19:34:15 [WARN] Collecting statistics
19:34:15 [WARN]  ex is: zero
19:34:15 [WARN]  ex is: (succ zero)
19:34:15 [WARN]  ex is: (succ (succ zero))
19:34:15 [WARN] Running exploration without proof mode, but goals were given
19:34:15 [WARN] Proof mode: true
19:34:15 [WARN]  ex is: zero
19:34:15 [WARN]  ex is: (succ zero)
19:34:15 [WARN]  ex is: (succ (succ zero))
19:34:15 [WARN] Running exploration without proof mode, but goals were given
19:34:15 [WARN] Proof mode: true
19:34:15 [WARN] proved: (leq zero ?ts_ph_Nat_0) => true
19:34:15 [WARN] proved: (qfac ?ts_ph_Nat_0 zero) => zero
19:34:15 [WARN] proved: (mult ?ts_ph_Nat_0 zero) => (qfac ?ts_ph_Nat_0 zero)
19:34:15 [WARN] proved: (qfac ?ts_ph_Nat_0 zero) => (mult ?ts_ph_Nat_0 zero)
19:34:15 [WARN] proved: (plus ?ts_ph_Nat_0 zero) => ?ts_ph_Nat_0
19:35:41 [WARN] proved: (fac (succ ?ts_ph_Nat_0)) => zero
19:35:41 [WARN] proved: (leq ?ts_ph_Nat_0 zero) => (leq ?ts_ph_Nat_0 (fac ?ts_ph_Nat_0))
19:35:41 [WARN] proved: (leq ?ts_ph_Nat_0 (fac ?ts_ph_Nat_0)) => (leq ?ts_ph_Nat_0 zero)
19:35:41 [WARN] proved: (plus ?ts_ph_Nat_0 (succ zero)) => (succ ?ts_ph_Nat_0)
19:35:41 [WARN] proved: (succ ?ts_ph_Nat_0) => (plus ?ts_ph_Nat_0 (succ zero))
19:35:41 [WARN] proved: (fac ?ts_ph_Nat_0) => (fac (plus ?ts_ph_Nat_0 ?ts_ph_Nat_0))
19:35:41 [WARN] proved: (fac (plus ?ts_ph_Nat_0 ?ts_ph_Nat_0)) => (fac ?ts_ph_Nat_0)
19:35:41 [WARN] proved: (fac ?ts_ph_Nat_0) => (qfac ?ts_ph_Nat_0 (fac ?ts_ph_Nat_0))
19:35:41 [WARN] proved: (qfac ?ts_ph_Nat_0 (fac ?ts_ph_Nat_0)) => (fac ?ts_ph_Nat_0)
19:35:42 [WARN] generalized case_split proof as: (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_2)) -- (plus (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1) ?ts_ph_Nat_2)
19:35:42 [WARN] proved: (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_2)) => (plus (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1) ?ts_ph_Nat_2)
19:35:42 [WARN] proved: (plus (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1) ?ts_ph_Nat_2) => (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_2))
19:35:42 [WARN] proved: (qfac (succ ?ts_ph_Nat_0) (succ ?ts_ph_Nat_1)) => (qfac (succ ?ts_ph_Nat_0) ?ts_ph_Nat_0)
19:35:43 [WARN] proved: (qfac (succ ?ts_ph_Nat_0) (succ ?ts_ph_Nat_1)) => zero
19:35:43 [WARN] proved: (qfac (succ ?ts_ph_Nat_0) (succ ?ts_ph_Nat_1)) => (qfac ?ts_ph_Nat_0 ?ts_ph_Nat_0)
19:35:43 [WARN] proved: (mult ?ts_ph_Nat_1 (qfac ?ts_ph_Nat_0 ?ts_ph_Nat_0)) => zero
19:35:43 [WARN] proved: (mult ?ts_ph_Nat_1 (qfac ?ts_ph_Nat_0 ?ts_ph_Nat_0)) => (qfac ?ts_ph_Nat_0 ?ts_ph_Nat_0)
19:35:43 [WARN] proved: (qfac ?ts_ph_Nat_0 (succ ?ts_ph_Nat_1)) => (mult (fac ?ts_ph_Nat_0) (succ ?ts_ph_Nat_1))
19:35:43 [WARN] proved: (mult (fac ?ts_ph_Nat_0) (succ ?ts_ph_Nat_1)) => (qfac ?ts_ph_Nat_0 (succ ?ts_ph_Nat_1))
19:35:44 [WARN] Found all lemmas
