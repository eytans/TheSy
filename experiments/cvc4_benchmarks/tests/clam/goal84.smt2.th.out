19:34:15 [0m[33m[WARN] [0mCollecting statistics
(leq ?__x0 ?__y1) => (or (= ?__x0 ?__y1) (less ?__x0 ?__y1))
(or (= ?__x0 ?__y1) (less ?__x0 ?__y1)) => (leq ?__x0 ?__y1)
(plus zero ?n) => ?n
(plus (succ ?n) ?m) => (succ (plus ?n ?m))
(succ (plus ?n ?m)) => (plus (succ ?n) ?m)
(mult zero ?n) => zero
(mult (succ ?n) ?m) => (plus (mult ?n ?m) ?m)
(plus (mult ?n ?m) ?m) => (mult (succ ?n) ?m)
(fac zero) => (succ zero)
(succ zero) => (fac zero)
(fac (succ ?n)) => (mult (fac ?n) ?n)
(mult (fac ?n) ?n) => (fac (succ ?n))
(qfac zero ?n) => ?n
(qfac (succ ?n) ?m) => (qfac ?n (mult ?m ?n))
(qfac ?n (mult ?m ?n)) => (qfac (succ ?n) ?m)
19:34:15 [0m[33m[WARN] [0m ex is: zero
19:34:15 [0m[33m[WARN] [0m ex is: (succ zero)
19:34:15 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:34:15 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:34:15 [0m[33m[WARN] [0mProof mode: true
19:34:15 [0m[33m[WARN] [0m ex is: zero
19:34:15 [0m[33m[WARN] [0m ex is: (succ zero)
19:34:15 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:34:15 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:34:15 [0m[33m[WARN] [0mProof mode: true
19:34:15 [0m[33m[WARN] [0mproved: (leq zero ?ts_ph_Nat_0) => true
19:34:15 [0m[33m[WARN] [0mproved: (qfac ?ts_ph_Nat_0 zero) => zero
19:34:15 [0m[33m[WARN] [0mproved: (mult ?ts_ph_Nat_0 zero) => (qfac ?ts_ph_Nat_0 zero)
19:34:15 [0m[33m[WARN] [0mproved: (qfac ?ts_ph_Nat_0 zero) => (mult ?ts_ph_Nat_0 zero)
19:34:15 [0m[33m[WARN] [0mproved: (plus ?ts_ph_Nat_0 zero) => ?ts_ph_Nat_0
19:35:41 [0m[33m[WARN] [0mproved: (fac (succ ?ts_ph_Nat_0)) => zero
19:35:41 [0m[33m[WARN] [0mproved: (leq ?ts_ph_Nat_0 zero) => (leq ?ts_ph_Nat_0 (fac ?ts_ph_Nat_0))
19:35:41 [0m[33m[WARN] [0mproved: (leq ?ts_ph_Nat_0 (fac ?ts_ph_Nat_0)) => (leq ?ts_ph_Nat_0 zero)
19:35:41 [0m[33m[WARN] [0mproved: (plus ?ts_ph_Nat_0 (succ zero)) => (succ ?ts_ph_Nat_0)
19:35:41 [0m[33m[WARN] [0mproved: (succ ?ts_ph_Nat_0) => (plus ?ts_ph_Nat_0 (succ zero))
19:35:41 [0m[33m[WARN] [0mproved: (fac ?ts_ph_Nat_0) => (fac (plus ?ts_ph_Nat_0 ?ts_ph_Nat_0))
19:35:41 [0m[33m[WARN] [0mproved: (fac (plus ?ts_ph_Nat_0 ?ts_ph_Nat_0)) => (fac ?ts_ph_Nat_0)
19:35:41 [0m[33m[WARN] [0mproved: (fac ?ts_ph_Nat_0) => (qfac ?ts_ph_Nat_0 (fac ?ts_ph_Nat_0))
19:35:41 [0m[33m[WARN] [0mproved: (qfac ?ts_ph_Nat_0 (fac ?ts_ph_Nat_0)) => (fac ?ts_ph_Nat_0)
19:35:42 [0m[33m[WARN] [0mgeneralized case_split proof as: (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_2)) -- (plus (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1) ?ts_ph_Nat_2)
19:35:42 [0m[33m[WARN] [0mproved: (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_2)) => (plus (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1) ?ts_ph_Nat_2)
19:35:42 [0m[33m[WARN] [0mproved: (plus (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1) ?ts_ph_Nat_2) => (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_2))
19:35:42 [0m[33m[WARN] [0mproved: (qfac (succ ?ts_ph_Nat_0) (succ ?ts_ph_Nat_1)) => (qfac (succ ?ts_ph_Nat_0) ?ts_ph_Nat_0)
19:35:43 [0m[33m[WARN] [0mproved: (qfac (succ ?ts_ph_Nat_0) (succ ?ts_ph_Nat_1)) => zero
19:35:43 [0m[33m[WARN] [0mproved: (qfac (succ ?ts_ph_Nat_0) (succ ?ts_ph_Nat_1)) => (qfac ?ts_ph_Nat_0 ?ts_ph_Nat_0)
19:35:43 [0m[33m[WARN] [0mproved: (mult ?ts_ph_Nat_1 (qfac ?ts_ph_Nat_0 ?ts_ph_Nat_0)) => zero
19:35:43 [0m[33m[WARN] [0mproved: (mult ?ts_ph_Nat_1 (qfac ?ts_ph_Nat_0 ?ts_ph_Nat_0)) => (qfac ?ts_ph_Nat_0 ?ts_ph_Nat_0)
19:35:43 [0m[33m[WARN] [0mproved: (qfac ?ts_ph_Nat_0 (succ ?ts_ph_Nat_1)) => (mult (fac ?ts_ph_Nat_0) (succ ?ts_ph_Nat_1))
19:35:43 [0m[33m[WARN] [0mproved: (mult (fac ?ts_ph_Nat_0) (succ ?ts_ph_Nat_1)) => (qfac ?ts_ph_Nat_0 (succ ?ts_ph_Nat_1))
19:35:44 [0m[33m[WARN] [0mFound all lemmas
done in 88294
